PWGCF/FemtoDream/FemtoDreamEventHisto.h

```diff
@@ -47,8 +49,10 @@ class FemtoDreamEventHisto
   {
     if (mHistogramRegistry) {
       mHistogramRegistry->fill(HIST("Event/zvtxhist"), col.posZ());
-      mHistogramRegistry->fill(HIST("Event/MultNTracksPV"), col.multNtrPV());
       mHistogramRegistry->fill(HIST("Event/MultV0M"), col.multV0M());
+      mHistogramRegistry->fill(HIST("Event/MultNTracksPV"), col.multNtrPV());
+      mHistogramRegistry->fill(HIST("Event/MultNTracksWithTPC"), col.multNtrWithTPC());
+      mHistogramRegistry->fill(HIST("Event/MultNTracklets"), col.multNtrlets());
```

Comment:
Are you analyzing for different multiplicity estimators?
It  looks to me that you are storing everything and deciding what to use later
This is not a good design practice